src/query/test/block.go

```diff
@@ -228,3 +234,21 @@ func GenerateValuesAndBounds(
 
 	return values, bounds
 }
+
+// MustMakeTags creates tags given that the number of args is even.
+func MustMakeTags(tag ...string) models.Tags {
```

Comment:
Hm doesn't this already appear in `block_test.go` somewhere in this PR already?

Reply:
Yeah managed to move it to `block` package

Reply:
Hm, can we use this method instead of redeclaring this method in `block/empty_test.go`?

Seems a private version is declared there:
```go
func mustMakeTags(tag ...string) models.Tags {
	if len(tag)%2 != 0 {
		panic("must have even tag length")
	}

 	tagLength := len(tag) / 2
	t := models.NewTags(tagLength, models.NewTagOptions())
	for i := 0; i < tagLength; i++ {
		t = t.AddTag(models.Tag{
			Name:  []byte(tag[i*2]),
			Value: []byte(tag[i*2+1]),
		})
	}

 	return t
}
```